python/lsst/cp/pipe/ptc/astierCovFitParameters.py

```diff
@@ -23,6 +23,8 @@
 
 import numpy as np
 
+__all__ = []
+
```

Comment:
Really? No exported symbols at all?

Reply:
The only class used elsewhere is `FitParameters`, and that's in `astierCovPtcFit.py`, which exports only `CovFit`.  I think once we've fully confirmed that the PTC and its downstream products (brighter-fatter kernel, linearity) are generating what we want, this entire part of the code will be refactored.

Reply:
Then I suggest putting `FitParameters` in the list, so that somebody looking at the file can tell which of these components is intended for external use. Making that explicit might well help with the refactor, too.